### DISCUSSION TITLE
MongoDB/Mongoose -> populate

### DISCUSSION THREAD

**Discussion by tomsmulski** [neuefische/questions#26] [Web] [answered]
Ich hab Google genutzt, andere gefragt und Google genutzt.

Google konnte mir nicht helfen,

ich habe hier ein denk Fehler denke ich mal, ich weiß aber nicht wo.

Ziel -> ich möchte eine 'populate' zur einer anderen Collection aufbauen die wie ein sql join arbeitet. 

danke für im voraus

![image](https://user-images.githubusercontent.com/77203530/181245855-355eb0e7-d472-4cd4-ac76-e0ba11047238.png)

**Comment by actyralf** [comment, marked as answer]
`populate()` bekommt als Parameter **den Feldnamen**, den Du "populaten" möchtest, in diesem Fall also `populate("buildMaterials")`
